prom2json.go

```diff
@@ -112,7 +112,6 @@ func makeHistogram(m *dto.Metric) Histogram {
 	hist := Histogram{
 		Labels:      makeLabels(m),
 		TimestampMs: makeTimestamp(m),
-		Count:       fmt.Sprint(dtoH.GetSampleCount()),
```

Comment:
Even classic histograms can be float histograms, so we need to treat this differently for both classic and native histograms.

So what we have to do is to first check `dtoH.GetSampleCountFloat()`. If that's > 0, take it. Otherwise, take `dtoH.GetSampleCount()`.

Reply:
ahh I didn't know that. I guess we need the same in pushgateway too :)